### ISSUE TITLE
Ability to control length of subtitles with Leopard

### ISSUE THREAD

**Issue by SammyJammy32** [Picovoice/leopard#164]
I'm using Leopard to try and create subtitles for a project, the problem is that each subtitle has way too many words and is way too long. I don't see any way to control this and I'm getting sections of subtitles that are too wordy

**Comment by kenarsa** [comment]
you can by changing `lenght_limit` [here](https://github.com/Picovoice/leopard/blob/master/demo/python-subtitle/main.py#L64).
